apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -1220,6 +1220,26 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 		return this.col.aggregate(params, { readPreference: readSecondaryPreferred() });
 	}
 
+	findPaginatedRoomsByVisitorsIdsAndSource({
+		visitorsIds,
+		source,
+		options = {},
+	}: {
+		visitorsIds: string[];
+		source?: string;
+		options: FindOptions;
```

Comment:
Are options required here? 🤔 

If so, let's reverse the params order, `source` is not required.